pkg/wasm/convert.go

```diff
@@ -215,16 +217,34 @@ func convertWasmConfigFromRemoteToLocal(ec *core.TypedExtensionConfig, wasmHTTPF
 	if remote.GetHttpUri().Timeout != nil {
 		timeout = remote.GetHttpUri().Timeout.AsDuration()
 	}
-	// ec.Name is resourceName.
-	// https://github.com/istio/istio/blob/9ea7ad532a9cc58a3564143d41ac89a61aaa8058/pilot/pkg/networking/core/v1alpha3/extension/wasmplugin.go#L103
-	f, err := cache.Get(httpURI.GetUri(), GetOptions{
-		Checksum:        remote.Sha256,
-		ResourceName:    ec.Name,
-		ResourceVersion: resourceVersion,
-		RequestTimeout:  timeout,
-		PullSecret:      pullSecret,
-		PullPolicy:      pullPolicy,
-	})
+
+	var (
+		f        string
+		err      error
+		attempts = 0
+	)
+
+	b := backoff.NewExponentialBackOff(backoff.DefaultOption())
+	for attempts < cacheMaxTry {
```

Comment:
Suggest using RetryWithContext(), and may add MaxRetries in that lib

Reply:
In this case, only retry if file exists